### PR TITLE
Project: Testing practice

### PR DESCRIPTION
**Pull request by lucasmazo32**

### CODE REVIEW

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @lucasmazo32 @mkezmkez 
Nice job done :smile: 
However, there are some requirements that should be met.

- As indicated [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/caesar_requirement.png), for the Ceasar Cipher you should write 3 tests, but I just saw one. Please, complete it.

Kindly make the suggested changes and submit a new code review request.
Thanks

Mael FOSSO

**Comment by lucasmazo32** [comment]
Thanks Mael.

Added more tests and actually notice some silent bugs. Already fixed them. 

Lucas

File: src/caesarCipher.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    19: 's',
+    20: 't',
+    21: 'u',
+    22: 'v',
+    23: 'w',
+    24: 'x',
+    25: 'y',
+    26: 'z',
+  };
+
```
Comment (shloch) [inline src/caesarCipher.js:30]: this project is about testing, but this code could be generated with a loop with less lines :)

**Review by shloch — APPROVED** [review]
## hello @mkezmkez , @mkezmkez 

GOOD To GO....tests runs smoothly
Screen Shot 2020-04-10 at 10 07 04 PM

For better organization, it would be better that each module has it's own test file..... If you could do that organization before merging, that would be good.

Such that we have something like
`calculator.js` && `calculator.test.js`

## THUMBS UP...Happy coding

**Pull request merged**